feature-libs/asm/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@spartacus/asm",
+  "version": "3.0.0-next.6",
+  "description": "ASM feature library for Spartacus",
+  "homepage": "https://github.com/SAP/spartacus",
+  "keywords": [
+    "spartacus",
+    "framework",
+    "storefront",
+    "asm"
+  ],
+  "scripts": {
+    "prepare:schematics": "yarn install",
+    "clean:schematics": "yarn prepare:schematics && rimraf \"schematics/**/*.js\" \"schematics/**/*.js.map\" \"schematics/**/*.d.ts\"",
+    "build:schematics": "yarn clean:schematics && tsc -p ./tsconfig.schematics.json",
+    "test:schematics": "yarn --cwd ../../projects/schematics/ run clean && yarn clean:schematics && jest --config ./jest.schematics.config.js"
```

Comment:
We need to disable schematics temporarily until the schematics are update to support the new storefront app module import structure.